app/furniture/marketplace/tags_controller.rb

```diff
@@ -12,6 +12,22 @@ def new
       authorize(mtag)
     end
 
+    def edit
+      authorize(mtag)
+    end
+
+    def update
+      if authorize(mtag).update(mtag_params)
+        if request.xhr?
+          render json: {}, status: :ok
+        else
+          redirect_to marketplace.location(child: :tags)
+        end
```

Comment:
Should I be using `respond_to |format|...` here? I tried by but for some reason after drag/drop I was still getting into the html block and it would try to redirect to `../<tag_id>/tags` incorrectly.

```ruby
# Example that redirects even for the ajax call 🤔
respond_to do |format|
	format.json { head :ok}
	format.html do 
		redirect_to marketplace.location(child: :tags)
	end
end 
```

Reply:
It depends on how the drag/drop request gets sent, I can't remember exactly how rails decides something is a "JSON" format, but I believe it's an intersection of xhr + content-type; which may not be being set by default on  `@stimulus-components/sortable`

Reply:
I'll double check the headers. I definitely don't mind `request.xhr?`.

Reply:
Ah, looks like Stimulus Sortable is actually sending an ajax request but with `Content-Type` as `multipart/form-data`.